prow/plugins/config.go

```diff
@@ -731,11 +731,12 @@ func (c *Configuration) setDefaults() {
 		c.CherryPickUnapproved.BranchRegexp = `^release-.*$`
 	}
 	if c.CherryPickUnapproved.Comment == "" {
-		c.CherryPickUnapproved.Comment = `This PR is not for the master branch but does not have the ` + "`cherry-pick-approved`" + `  label. Adding the ` + "`do-not-merge/cherry-pick-not-approved`" + `  label.
+		c.CherryPickUnapproved.Comment = `This cherry pick PR is for a release branch and has not yet been approved by the Patch Release Team.
+Adding the ` + "`do-not-merge/cherry-pick-not-approved`" + ` label.
 
-To approve the cherry-pick, please ping the *kubernetes/patch-release-team* in a comment when ready.
+To merge this cherry pick, please ping the ` + "`@kubernetes/patch-release-team`" + ` in a comment, **after** it has been approved by the relevant OWNERS.
```

Comment:
This will still trigger pings when the many people use email to reply to this comment